cmd/util/format.go

```diff
@@ -36,6 +48,13 @@ func Printf(msg string, a ...interface{}) {
 	}
 }
 
+// SafePrintf is a thread-safe wrapper to Printf
```

Comment:
This would probably be fine as the default implementation, locking is relatively cheap and we always want serialized output.

Reply:
Meh, let's leave it as-is. Not that much more complicated, and it could be useful to export the locks for bulk writes in the future.